packages/components/src/components/hds/tabs/index.ts

```diff
@@ -131,33 +163,33 @@ export default class HdsTabsIndexComponent extends Component {
   }
 
   @action
-  didUpdateTab(tabIndex, isSelected) {
+  didUpdateTab(tabIndex: number, isSelected?: boolean) {
     if (isSelected) {
       this.selectedTabIndex = tabIndex;
     }
     this.setTabIndicator();
   }
 
   @action
-  willDestroyTab(element) {
+  willDestroyTab(element: HTMLButtonElement) {
     this.tabNodes = this.tabNodes.filter((node) => node.id !== element.id);
     this.tabIds = this.tabIds.filter((tabId) => tabId !== element.id);
   }
 
   @action
-  didInsertPanel(element, panelId) {
+  didInsertPanel(element: HTMLElement, panelId: string) {
```

Comment:
no biggie, but do you think it would make sense to have this signature come from the `HdsTabsPanelSignature['Args']['didInsertNode']` somehow? @alex-ju thoughts? I'm overthinking it?

(similar for other functions here)

Reply:
I'd rather define shared types from more of a top-down approach if that makes sense. Curious for @alex-ju thoughts as well.

Reply:
> I'd rather define shared types from more of a top-down approach if that makes sense.

That sounds good to me and keeps our types less coupled